src/main/kotlin/graphics/scenery/volumes/DummyVolume.kt

```diff
@@ -0,0 +1,69 @@
+package graphics.scenery.volumes
+
+import bdv.tools.brightness.ConverterSetup
+import graphics.scenery.*
+import graphics.scenery.net.Networkable
+import java.lang.IllegalArgumentException
+
+class DummyVolume(val counterStart : Int = 0) : DefaultNode("DummyVolume"),HasTransferFunction {
```

Comment:
Is there any reason why this class needs to exist, instead of using a regular `Volume` with a `NullSource` instead?